sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -564,6 +564,10 @@ def test_pairwise_distances_reduction_is_usable_for():
     assert not BaseDistancesReductionDispatcher.is_usable_for(
         X_csr, Y_csr, metric="sqeuclidean"
     )
+    # TODO: add this assertion once #24542 and #24715 are merged.
+    # assert BaseDistancesReductionDispatcher.is_usable_for(
+    #     X_csr, Y_csr, metric="euclidean"
+    # )
```

Comment:
I don't understand why this can be added right away.

Reply:
Ah yes, I meant to remove the `not` of the previous assertion and to comment it.

This one can be removed.